R1SN003/wrappers/camera/pointcloudsensor_ros.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE robot PUBLIC "-//YARP//DTD yarprobotinterface 3.0//EN" "http://www.yarp.it/DTD/yarprobotinterfaceV3.0.dtd">
+
+<device xmlns:xi="http://www.w3.org/2001/XInclude" name="pointCloudWrapR" type="RGBDToPointCloudSensor_nws_ros">
+    <param name="period"> 0.033 </param>
```

Comment:
Hi @ste93 

I've seen that this period is given in seconds (I think YARP device drivers will comply w/ seconds in the future) whereas the others are given in milliseconds (standard way as of now).

Is this mixed-mode intended?

If so, I'll be eager to merge the PR.